frontend/src/components/InputForm.tsx

```diff
@@ -46,8 +74,13 @@ export default function InputForm(props: Props) {
         setPrice(event.target.value)
     }
 
+
+
+
+
+
     return (
-        <form onSubmit={handleSubmit}>
+        <form onSubmit={props.gif?handleSubmit:handleSubmit}>
```

Comment:
Das `if` und `else` von deiner Bedingung ergibt das gleiche Ergebnis. Es ist immer handleSubmit. Wenn du, egal ob ein `gif` existiert immer die `handleSubmit` funktion benutzen möchtest, kannst du das auch so schreiben:
```suggestion
        <form onSubmit={props.gif && handleSubmit}>
```